lib/site-inspector/endpoint.rb

```diff
@@ -74,8 +69,9 @@ def up?
       response && response_code.start_with?("2") || response_code.start_with?("3")
     end
 
-    def down?
-      !up?
+    # Does the server respond at all?
+    def responds?
+       response.code != 0 && !timed_out?
     end
```

Comment:
Would it be worth adding an alias here so that we have one less breaking change?